plugin-server/src/cdp/services/managers/persons-manager.service.ts

```diff
@@ -0,0 +1,36 @@
+import LRUCache from 'lru-cache'
+
+import { CyclotronPerson } from '~/cdp/types'
+
+import { Hub } from '../../../types'
+
+const PERSONS_CACHE_AGE_MS = 60 * 1000 // 1 minute
+const PERSONS_CACHE_MAX_SIZE = 100_000 // 100k entries
+
+export class PersonsManagerService {
+    private personsCache: LRUCache<string, CyclotronPerson>
+
+    constructor(private hub: Hub) {
+        this.personsCache = new LRUCache({ max: PERSONS_CACHE_MAX_SIZE, maxAge: PERSONS_CACHE_AGE_MS })
```

Comment:
Generally I don't think any of this will work. 

We can't use a cache really as we need to make sure the Person is loaded fresh for an event (if the next event caused a person merge we want the latest info, not info from 10mins ago.

Also, this will do 100s of individual requests to the DB which is also nasty. We should be doing this lazyloader style and doing one fetch for multiple persons. 

I'll make the changes here to try out an alternative approach